pkg/controller/daemon/daemoncontroller_test.go

```diff
@@ -135,7 +135,7 @@ func addPods(podStore cache.Store, nodeName string, label map[string]string, num
 }
 
 func newTestController() (*DaemonSetsController, *controller.FakePodControl) {
-	clientset := clientset.NewForConfigOrDie(&restclient.Config{Host: "", ContentConfig: restclient.ContentConfig{GroupVersion: testapi.Default.GroupVersion()}})
```

Comment:
Please stop spamming me with email updates about this, The mute link does not work. Someone incorrectly mentioned me in some comment now I get 20-30 email spam messages a day from this thread. Someone remove me.